### PR TITLE
Skill - SteadyBrawler

### PR DESCRIPTION
**Pull request by JesterWizard**
+25% battle attack if this unit can double, otherwise -25%.

### CODE REVIEW

File: Wizardry/Core/BattleSys/Source/PreBattleCalc.c [hunk 1]
```diff
@@ -1207,6 +1207,15 @@ void PreBattleCalcSkills(struct BattleUnit * attacker, struct BattleUnit * defen
                 attacker->battleSpeed += SKILL_EFF0(SID_AirRaidSpeed);
             break;
 #endif
+
+#if (defined(SID_SteadyBrawler) && (COMMON_SKILL_VALID(SID_SteadyBrawler)))
+        case SID_SteadyBrawler:
+            if (attacker->battleSpeed - defender->battleSpeed >= BATTLE_FOLLOWUP_SPEED_THRESHOLD)
```
Comment (MokhaLeee) [inline Wizardry/Core/BattleSys/Source/PreBattleCalc.c:1213]: just as what I have told， it is bad to only check on speed

Comment (JesterWizard) [reply Wizardry/Core/BattleSys/Source/PreBattleCalc.c:1213]: Then do I move the check to `battleOrder.c` and see if a fighter related skill has been applied first?

**Comment by MokhaLeee** [comment]
It seems that it is a little troublesome to adapt to the existing system. I tend to merge skill info first.

**Comment by JesterWizard** [comment]
Okay I'll let you do it then. Skill effect has been removed.

**Pull request merged**